content/en/blog/2023/any-metric-receiver.md

```diff
@@ -0,0 +1,470 @@
+2023-11-24T12:52:51.341+0100	info	exporter@v0.89.0/exporter.go:275	Development component. May change in the future.	{"kind": "exporter", "data_type": "metrics", "name": "debug"}
+2023-11-24T12:52:51.341+0100	info	service@v0.89.0/service.go:143	Starting otelcol-any-metric...	{"Version": "1.0.0", "NumCPU": 10}
+2023-11-24T12:52:51.341+0100	info	extensions/extensions.go:34	Starting extensions...
+2023-11-24T12:52:51.342+0100	info	service@v0.89.0/service.go:169	Everything is ready. Begin running and processing data.
+```
+
+{{% alert title="Note" color="secondary" %}}
+
+For testing, you can utilize the
+[OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib)
+distribution, which includes all available receivers. However, in a production
+setting, it's recommended to
+[construct your own Collector](/docs/collector/custom-collector/) using the
+OpenTelemetry Collector Builder
+([`ocb`](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder)).
+Here's a suggested configuration:
+
+```yaml
+dist:
+  name: otelcol-any-metric
+  description: Custom OpenTelemetry Collector for receiving any kind of metric
+  output_path: ./
+
+exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.89.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.89.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.89.0
+
+processors:
+  - gomod:
+      github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor
+      v0.89.0
+
+receivers:
+  - gomod: go.opentelemetry.io/collector/receiver/otlpreceiver v0.89.0
+  - gomod:
+      github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver
+      v0.89.0
+```
+
+{{% /alert %}}
+
+With the OpenTelemetry Collector operational, open a secondary shell and
+transmit your metric to it:
+
+```shell
+./ssl_check.sh opentelemetry.io | nc 127.0.0.1 8080
+```
+
+The
+[Debug Exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/debugexporter)
+will display the metric on the console for you:
+
+```log
+2023-11-24T12:54:51.369+0100	info	ResourceMetrics #0
+Resource SchemaURL:
+ScopeMetrics #0
+ScopeMetrics SchemaURL:
+InstrumentationScope
+Metric #0
+Descriptor:
+     -> Name: tls.server.not_after.time_left
+     -> Description:
+     -> Unit:
+     -> DataType: Gauge
+NumberDataPoints #0
+Data point attributes:
+     -> unit: Str(s)
+StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
+Timestamp: 2023-11-24 11:54:51 +0000 UTC
+Value: 4356471
+	{"kind": "exporter", "data_type": "metrics", "name": "debug"}
+```
+
+That's it! You can implement the same technique with any other shell script you
+have that reports custom metrics.
+
+## Fine Tuning with the Transform Processor
```

Comment:
```suggestion
## Fine tuning with the Transform processor
```

As all other headings are using sentence case

Reply:
```suggestion
## Fine tuning with the Transform Processor
```

Will apply the Tuning to tuning, but I use Transform Processor in uppercase since this is also how it is used throughout the text and in the collector repos.